server/dash/dashboards/neighborhood.py

```diff
@@ -30,59 +32,97 @@
 
 def populate_options():
     values = []
-    for i in df.sort_values('council_name').council_name.unique():
-        values.append(
-            {
-                'label': i,
-                'value': i
-            }
-        )
+    for i in df.sort_values("council_name").council_name.unique():
+        values.append({"label": i, "value": i})
     return values
 
 
 # Define callback to update graph
 @app.callback(
-    Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    Output("graph1", "figure"),
+    [Input("council_list", "value"), Input("data_type", "value")],
 )
-def update_figure(selected_council):
+def update_figure(selected_council, selected_timeframe):
+    """Creates comparison graph
+
+    Args:
+        selected_council:
+          A string representing the council selected from the drop down menu.
```

Comment:
Did the formatter put "A string..." on a new line? The arg and the description are supposed to be on the same line. Ditto for "Returns" section.

Reply:
I think it did. I changed it to reflect your comment